app/models/user.rb

```diff
@@ -3,4 +3,11 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
   enum gender: { male: 0, female: 1 }
+  accepts_nested_attributes_for :interests, reject_if: :all_blank, allow_destroy: true
+
+  private
+  def self.create_with_password(attr={})
```

Comment:
No i was check this second time and you have right i don't need default. When i create this it was't work meaby some small bug ?? :)